cli/main.go

```diff
@@ -40,8 +39,20 @@ func main() {
 
 	app.Commands = []cli.Command{
 		{
-			Name:    "up",
-			Aliases: []string{"apply"},
+			Name:    "apply",
+			Aliases: []string{},
+			Usage:   "Apply a modules in a path",
```

Comment:
s/modules/module/

(This error occurs a couple times.)

Reply:
Thanks, corrected.